### DISCUSSION TITLE
中西医结合的诊疗方案模型

### DISCUSSION THREAD

**Discussion by Shengshenlan** [InternLM/Tutorial#339] [project]
## 项目成员
Shengshenlan
Egebr1t
siuser
Charles
NANDSS1
成员已满

## 项目详情 
![image](https://github.com/InternLM/tutorial/assets/57640594/ab1ee7a2-a2e8-4bfb-9334-3fd9cfc486ad)

文档链接：https://fa6qwwph5la.feishu.cn/wiki/J7QnwK9kziMC4IkddSKclAKfnJe

**Comment by scchy** [comment]
你好， 还招成员么？

我的进阶部署实践如下：
- HW3 langchain-部署: 
  - Github: https://github.com/scchy/LLM_CookingAssistant
  - openxlab应用： https://openxlab.org.cn/apps/detail/Scchy/LLM_CookingAssistant
- HW4 xtuner-adpater-merge部署: 
  - Github: https://github.com/scchy/LLM_medQA
  - openxlab应用： https://openxlab.org.cn/apps/detail/Scchy/LLM_medQA

**Reply by Shengshenlan** [reply]
谢谢你，但我们成员已满
